RestingStateStats/RestingStateStats.sh

```diff
@@ -968,7 +968,7 @@ main()
 			mGlobalPath="${HCPPIPEDIR}/global/matlab"
 			mFslPath="${FSLDIR}/etc/matlab"
 
-			matlabCode="addpath '$mPath'; addpath '$mGlobalPath'; addpath '$mFslPath'; RestingStateStats('${motionparameters}',${g_high_pass},${TR},'${ICAs}','${noise}','${CARET7DIR}/wb_command','${dtseries}','${bias}','${RssPrefix}','${g_dlabel_file}','${g_bc_mode}','${g_out_string}','${WM}','${CSF}');"
+			matlabCode="addpath '$mPath'; addpath '$HCPCIFTIRWDIR' addpath '$mGlobalPath'; addpath '$mFslPath'; RestingStateStats('${motionparameters}',${g_high_pass},${TR},'${ICAs}','${noise}','${CARET7DIR}/wb_command','${dtseries}','${bias}','${RssPrefix}','${g_dlabel_file}','${g_bc_mode}','${g_out_string}','${WM}','${CSF}');"
```

Comment:
In addition to path order consistency, existing command missing a semicolon between two `addpath`
```suggestion
			matlabCode="addpath '$mFslPath'; addpath '$HCPCIFTIRWDIR'; addpath '$mGlobalPath'; addpath '$mPath'; RestingStateStats('${motionparameters}',${g_high_pass},${TR},'${ICAs}','${noise}','${CARET7DIR}/wb_command','${dtseries}','${bias}','${RssPrefix}','${g_dlabel_file}','${g_bc_mode}','${g_out_string}','${WM}','${CSF}');"
```